src/docs/patterns/appbar.mdx

```diff
@@ -35,3 +37,37 @@ In some situations, you may want to present more information in an app bar, such
 <DemoCard repository={'search-bar'} angular react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
 A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+
+<Divider />
+
+# Design Specifications
+
+For additional instructions, refer to [Material Design's design specs on "App bars: top"](https://material.io/components/app-bars-top#specs).
```

Comment:
This is the same link as in the callout card above...should we move that down here? Do we need it in both places?

Reply:
Yes. I think we should not make our users scroll all the way to the top, but just offer them handy... Or, we can simply "import" the material design dimensions, but replacing the components with pxb app bars.

Reply:
I don't mind having the link in multiple places, but should we use the same link (card) style...or something similar to reinforce the idea of "Hey this goes to the Material documentation"?

Reply:
Changed to the callout card. Their deep link is messed up -- it should have taken people directly to the #Specs section

Reply:
Did you push that change? I'm still seeing the same link.